addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1174,6 +1174,8 @@ var SnippetEditor = Widget.extend({
         this.$body.addClass('move-important');
 
         this.$dropZones = this.$editable.find('.oe_drop_zone');
+        // Remove no-block zones.
+        this.$dropZones = this.$dropZones.not('[data-oe-sanitize="no_block"] .oe_drop_zone');
```

Comment:
Is there any reason to actually let the `[data-oe-sanitize="no_block"]` receive a `oe_drop_zone` in the first place?

Shouldn't we prevent such elements to receive dropzone, and so nothing should be filter out here (and other place in the PR)?
Maybe in `_insertDropzone` or in callers?

Reply:
Indeed what is done in `_insertDropZone` actually already prevents the other cases - I removed the other `.not` & `:not` s.